### PR TITLE
(pedia) fix border checkpoints and environmentalism policy description

### PR DESCRIPTION
**Pull request by MichaelCourtney**

### CODE REVIEW

**Comment by MichaelCourtney** [comment]
Its on draft because of last line.

"Reduces influence generation(-0.5)"

Influence generation per a planet?
per a colony/outpost?
How best to write this?

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -12582,14 +12582,14 @@ PLC_CHECKPOINTS
 Border Checkpoints
 
 PLC_CHECKPOINTS_DESC
-'''Decreases [[metertype METER_STEALTH]] of foreign ships and planets in [[metertype METER_SUPPLY]] by up to [[value PLC_CHECKPOINTS_MAX_STEALTH_MALUS]]. Ships and planets are considered foreign if not owned by or allied to this empire. Unowned planets and monsters are also foreign.
+'''Decreases [[metertype METER_STEALTH]] of foreign ships in [[metertype METER_SUPPLY]] by up to [[value PLC_CHECKPOINTS_MAX_STEALTH_MALUS]].
 
 The decrease depends on the time this policy is adopted, decreasing [[metertype METER_STEALTH]] by 1 per turn up to [[value PLC_CHECKPOINTS_MAX_STEALTH_MALUS]].
 
 Reduces [[metertype METER_INFLUENCE]] generation ([[value PLC_CHECKPOINTS_TARGET_INFLUENCE_FLAT]]).'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: `...on planets of the empire that adopts this policy` or similar?

Comment (MichaelCourtney) [reply default/stringtables/en.txt]: I'll write per a colony if the IP cost effect is removed from outposts. Charging IP on outposts is currently running into outposts not have PP/RP/IP growth/decay and inconsistent with other policies.

**Comment by MichaelCourtney** [comment]
Checkpoints description assumes this goes through:
https://github.com/freeorion/freeorion/pull/4892

Environmentalism description was missed here:
https://github.com/freeorion/freeorion/commit/6ca72d1e1ade696ca502a9cc32db1d9e79656975

**Pull request merged**